Caboodle/AppInfo/AppInfo.android.cs

```diff
@@ -43,5 +39,24 @@ static string GetBuild()
                 return info.VersionCode.ToString(CultureInfo.InvariantCulture);
             }
         }
+
+        static async Task<string> PlatformGetLatestVersionStringAsync()
+        {
+            var version = string.Empty;
+            var url = $"https://play.google.com/store/apps/details?id={GetPackageName()}";
```

Comment:
Android is a bit hacky as there is no official API, so we just ask the website. This may return "Multiple Versions" for apps that have multiple APKs